src/FunctionLoader.cs

```diff
@@ -57,6 +59,11 @@ internal static void SetupWellKnownPaths(FunctionLoadRequest request)
             var appLevelModulesPath = Path.Join(FunctionAppRootPath, "Modules");
             var workerLevelModulesPath = Path.Join(AppDomain.CurrentDomain.BaseDirectory, "Modules");
             FunctionModulePath = $"{appLevelModulesPath}{Path.PathSeparator}{workerLevelModulesPath}";
+            if (!string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable(AzureWebsiteInstanceId))
```

Comment:
Why not use `string.IsNullOrEmpty` as in the Host code?
https://github.com/fabiocav/azure-functions-host/blob/2fc6590888f6bcc38402eb80645701eb02a3f5ef/src/WebJobs.Script/Environment/EnvironmentExtensions.cs#L15-L18

Reply:
It's more of a question about whether it's needed. But I'm fine with the code as is.